Revise Shutdown and Draining section (#1512)
getsentry/sentry-docs: src/collections/_documentation/error-reporting/configuration/drain-example/python.md

if client is not None:
    client.close(timeout=2.0)
```

> After shutdown the current client cannot be used any more so make sure to only

Reviewer: ```suggestion
After shutdown, the current client cannot be used anymore, so make sure to
```

> do that right before you shut down the application.

Author: This is moved here from the global context, with one important edit: "the client" => "the **current** client".

> do that right before you shut down the application.

Reviewer: ```suggestion
do that before you shut down the application.
```

Alternatively, the `flush` method drains the event queue while keeping the

> client enabled for continued use.

Author: I would understand if @untitaker would rather omit this note. Thoughts?
